src/anemoi/graphs/nodes/attributes.py

```diff
@@ -125,11 +125,13 @@ def __init__(
         centre: np.ndarray = np.array([0, 0, 0]),
```

Comment:
I would say that it can make sense to split these node attributes into 2, given that the functionality is completely different for flat=True and flat=False. I can suggest some names such as `Earth/Globe/SphereAreaWeights`, and `2D/PlaneAreaWeight`.

Reply:
Agreed, will do